core/src/main/java/org/mule/routing/AbstractUntilSuccessfulProcessingStrategy.java

```diff
@@ -20,6 +25,7 @@
 {
 
     private UntilSuccessfulConfiguration untilSuccessfulConfiguration;
+    protected boolean requireSerializablePayload;
```

Comment:
if this is a field, then it should be final and indeally private. Otherwise, it could be a protected method